docs/readmes/proposals/p018_control_network_metrics.md

```diff
@@ -366,6 +366,32 @@ AGW capacity in terms of number of UEs, varying attach/detach rates, data path
 traffic. We will also aim to compensate statistically for the jitter in AGW
 capacity measurements.
 
+## Updates
```

Comment:
Super happy that you are updating the documentation. Thanks!

My concern with how we are updating the documentation is this (correct me if I'm wrong).  By putting the changes into an Updates subsection at the end of the document, people are going to build a model of your work as they read the document.  Quite possibly they won't finish reading the document (lets be real) and will walk away with incorrect assumptions until they realize otherwise.  Or best case, they reach this section and have to mentally "tear up" what they had learned above. 

If you want to preserve the Plan A for future posterity (is that why you are doing this?) Maybe re-order this and put `## Original Plan` at the bottom (or `## Alternatives Considered`) and move these updates inline above so the document reflects reality early? WDYT?

Reply:
I did want to preserve the original plan for posterity. You're right about people missing this or having to change their understanding of the design though. I'll re-order and bring the old stuff down as "Original Plan". Thanks!

Reply:
Done.